src/libtriton/arch/x86/x86Semantics.cpp

```diff
@@ -10481,6 +10483,85 @@ namespace triton {
         this->controlFlow_s(inst);
       }
 
+      void x86Semantics::shld_s(triton::arch::Instruction& inst) {
```

Comment:
Thanks for this function. The semantics is good and all my tests were successfully executed =). The coding style is not like others functions but it's not a big deal I will modify the coding style later.